applications/byte_patrol/urls.py

```
@@ -0,0 +1,8 @@
+from django.urls import path
```

Pylint (3.3.7)

[error] Plugin 'pylint_django' is impossible to load, is it installed ? ('No module named 'pylint_django'')

(E0013)